payments/card-payments/other-features.md

```diff
@@ -136,31 +136,31 @@ Content-Type: application/json
         "cardholder": {
             "firstName": "firstname/companyname",
             "lastName": "lastname",
-            "email": "string",
-            "msisdn": "string",
-            "homePhoneNumber": "string",
-            "workPhoneNumber": "string",
+            "email": "email",
+            "msisdn": "msisdn",
```

Comment:
Instead of `msisdn`, enter an actual phone number from our test data. The same goes for all other example values: Use test data instead.